akka-stream/src/main/scala/akka/stream/CompletionStrategy.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream
+
+import akka.annotation.DoNotInherit
+
+@DoNotInherit
+sealed trait CompletionStrategy
+
+case object CompletionStrategy {
+  case object Immediately extends CompletionStrategy
+  case object Draining extends CompletionStrategy
```

Comment:
this doesn't work from Java, probably easiest to change to `def` instead, similar to what is in `OverflowStrategy`

Reply:
I've pushed a rework